### PR TITLE
feat(files): shipping options custom component

### PR DESCRIPTION
**Pull request by kibo-amolpawar**

### CODE REVIEW

File: components/KiboShippingOptions.vue [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      <SfRadio
+        v-for="(shippingMethod, index) in shippingMethods"
+        :key="`${shippingMethod.name}_${index}`"
+        :name="shippingMethod.name"
+        :value="shippingMethod.value"
+        :label="shippingMethod.label"
+        :details="shippingMethod.details"
+        :required="shippingMethod.required"
+        class="sf-radio"
+        @change="selectShipping($event)"
```
Comment (kibo-kevinwatts) [inline components/KiboShippingOptions.vue]: event change should pass the shippingMethod object

File: components/KiboShippingOptions.vue [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  },
+  props: {
+    shippingMethods: {
+      type: Array,
+      default: () => [],
+    },
+  },
+  setup(_, context) {
+    const selectShipping = (selectedMethod: string) => {
+      context.emit("shippingMethod", selectedMethod)
```
Comment (kibo-kevinwatts) [inline components/KiboShippingOptions.vue]: set event name to "change"

File: components/KiboShippingOptions.vue [hunk 3]
```diff
@@ -0,0 +1,63 @@
+<template>
```
Comment (kibo-kevinwatts) [inline components/KiboShippingOptions.vue]: change "shipping" to "fulfillment" across this entire component.

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]
couple of name changes

File: components/KiboShippingOptions.vue [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+      context.emit("shippingMethod", selectedMethod)
+    }
+
+    return {
+      selectShipping,
+    }
+  },
+})
+</script>
+<style lang="scss">
```
Comment (kibo-sushant) [inline components/KiboShippingOptions.vue]: Apply Scoped

File: components/KiboShippingOptions.vue (hunk 1 above)
Comment (kibo-amolpawar) [reply components/KiboShippingOptions.vue]: passed the shipping object

File: components/KiboShippingOptions.vue (hunk 3 above)
Comment (kibo-amolpawar) [reply components/KiboShippingOptions.vue]: renamed.

File: components/KiboShippingOptions.vue (hunk 4 above)
Comment (kibo-amolpawar) [reply components/KiboShippingOptions.vue]: Overriding global CSS for this component so its removed.

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**